lib/direncrypt/fileops.py

```diff
@@ -1,14 +1,43 @@
 import os
+from direncrypt.util import printit
 
 class FileOps(object):
 
     @staticmethod
-    def delete_file(directory, filename):
-        """Try to delete file from filesystem."""
-        print("Deleting file on filesystem: {}".format(filename))
+    def delete_file(root_dir, file_name):
+        """Try to delete file from filesystem.
+        
+        returns true if successful, false otherwise
+        """
         try:
-            os.unlink(os.path.expanduser(os.path.join(directory, filename)))
-        except OSError as e:
-            print("Failed to delete {}: {}".format(filename, str(e)))
+            os.unlink(os.path.expanduser(os.path.join(root_dir, file_name)))
+        except OSError as ose:
+            printit("Failed to delete file {} : {}",file_name, str(ose))
             return False
         return True
+    
+    @staticmethod
+    def create_symlink(root_dir, link_name, target):
+        """Try to create a symlink pointing to target
+                
+        returns true if successful, false otherwise
+        """
+        try:
+            os.symlink(target, os.path.expanduser(os.path.join(root_dir, link_name)))
+        except Exception as e:
```

Comment:
Change `Exception` to `OSError` here as well.

Reply:
done